python-regression/tests/features/machine2/2_transaction_tests.feature

```diff
@@ -87,3 +87,106 @@ Feature: Test transaction confirmation
             | keys   | values | type          |
             | states | False  | boolListMixed |
 
+    Scenario: Valid value transfer bundle that doesnt affect ledger state
+        We want to ascertain that ledger state is always calculated correctly.
+        Even in the presence of a bundle that hadnales funds but without chaning address
+
+        Then "1" transaction is issued on "nodeA-m2" with:
+        |keys                   |values                     |type           |
+        |address                |TEST_ADDRESS               |staticValue    |
+        |value                  |0                          |int            |
+        |tag                    |ZERO9VALUE                 |string         |
+
+        Then a value transaction which does not move funds is generated referencing the previous transaction with:
+        |keys                   |values                     |type           |
+        |seed                   |THE_BANK                   |staticList     |
+        |value                  |100                        |int            |  
+        |tag                    |FAKE9VALUE                 |string         |
+
+        Then a transaction is issued referencing the previous transaction
+        |keys                   |values                     |type           |
+        |seed                   |THE_BANK                   |staticList     |
+        |address                |TEST_ADDRESS               |staticValue    |
+        |value                  |11                         |int            |
+        |tag                    |VALUE9TRANSACTION          |string         |
+
+        #In the default test, the latest sent index will be 52. The next milestone issued should be 53.
+        When a milestone is issued with index 53 and references:
+        |keys                   |values                     |type           |
+        |transactions           |previousTransaction        |responseValue  |
+
+        #Give the node time to solidify the milestone
+        And we wait "15" second/seconds
+        
+        Given "getBalances" is called on "nodeA-m2" with:
+        |keys                   |values                     |type           |
+        |addresses              |FAKE_SPEND_ADDRESSES       |staticList     |
+
+        Then the response for "getBalances" should return with:
+        |keys                   |values                     |type           |
+        |balances               |0                          |int            | 
```

Comment:
I am trying to read this test intentionally without looking at the issue

I don't really understand where `FAKE_SPEND_ADDRESSES` is coming from
Don't understand exactly what you are testing

Reply:
FAKE_SPEND_ADDRESSES is coming from staticList, which is static_vals.py, like all other static values.

What i'm testing is described in the issue ticket, basically sending nonexistant iota backand forth, causing 0 ledger changes after applying. Which is a valid case and should be allowed. Therefor we should test it :)

Reply:
I wasn't clear
I meant that by reading Gherkin, I see this value only mentioned once

I don't think I can understand why the actions you are doing before have a chance to mutate the balance on this address

Reply:
What if we immediately apply balance changes instead of only per bundle? BOOM test saves us as there is no balance